src/plugins/impl/fingerprint.plugin.ts

```diff
@@ -109,4 +118,12 @@ export class FingerprintPlugin implements IPlugin {
         });
         await this.bioAuthService.bulkSave({fingerprints});
     }
+
+    private async callExternalWalletCreate(externalIds: ExternalId[]): Promise<string> {
+        this.externalController.callExternalWalletCreate(externalIds[0].external_id);
```

Comment:
I think you need an `await` here. Otherwise it happens in the background and we'll return agentIds whether it succeeds or fails.

Separately, aren't we expecting to return the agentId from the result of the wallet create?